lib/sort-lines.coffee

```diff
@@ -16,30 +16,28 @@ module.exports =
         editor = atom.workspace.getActiveTextEditor()
         sortLinesInsensitive(editor)
 
-sortLines = (editor) ->
+sortTextLines = (editor, sorter) ->
   sortableRanges = RangeFinder.rangesFor(editor)
   sortableRanges.forEach (range) ->
     textLines = editor.getTextInBufferRange(range).split("\n")
-    textLines.sort (a, b) -> a.localeCompare(b)
+    textLines = sorter(textLines)
     editor.setTextInBufferRange(range, textLines.join("\n"))
 
+sortLines = (editor) ->
+   sortTextLines editor, (textLines) ->
+    textLines.sort (a, b) -> a.localeCompare(b)
+    return textLines
```

Comment:
Since `Array.prototype.sort` returns the array, you could remove this return statement.